bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/_rbdt.py

```diff
@@ -35,11 +35,24 @@ def Compute(self, x):
     # As fall-through we go to the original compute function and use the error-handling from cppyy
     return self._OriginalCompute(x)
 
+def RBDTInit(self, *args):
```

Comment:
```suggestion
def RBDTInit(self, *args, **kwargs):
```
I think you need to do also forward the keyword arguments

Reply:
Nice catch!